lib/membrane_live_web/plugs/auth.ex

```diff
@@ -6,28 +6,25 @@ defmodule MembraneLiveWeb.Plugs.Auth do
   import Plug.Conn
   alias MembraneLive.Tokens
   alias MembraneLiveWeb.FallbackController
+  alias MembraneLiveWeb.Helpers.TokenErrorInfo
 
+  @auth_header_key "authorization"
+
+  @spec init(any) :: any
   def init(default), do: default
 
   def call(conn, _default) do
-    with {"authorization", "Bearer " <> token} <- find_bearer(conn.req_headers),
-         {:ok, %{"user_id" => user_id}} <- Tokens.custom_decode(token) do
+    with {@auth_header_key, "Bearer " <> token} <-
+           find_header_value(conn.req_headers, @auth_header_key),
+         {:ok, %{"user_id" => user_id}} <- Tokens.auth_decode(token) do
       assign(conn, :user_id, user_id)
     else
       err ->
-        FallbackController.call(conn, get_error_info(err))
+        FallbackController.call(conn, TokenErrorInfo.get_error_info(err))
         |> halt()
```

Comment:
According to the style guide, you shouldn't use pipes only once.
https://github.com/christopheradams/elixir_style_guide#avoid-single-pipelines
```suggestion
        conn
        |> FallbackController.call(TokenErrorInfo.get_error_info(err))
        |> halt()
```

Reply:
The more you now 🌈⭐️✨